medusa/__main__.py

```diff
@@ -2070,6 +2078,51 @@ def shutdown(self, event):
             logging.shutdown()
             os._exit(0)  # TODO: Remove in another PR. There's no need for this one.
 
+    @staticmethod
+    def configure_unrar():
+        """Check if user has a valid UNRAR installation and set accordinly."""
+        try:
+            # If previously set from PATH or custom, try it first
+            if app.UNRAR_TOOL:
+                rarfile.custom_check(app.UNRAR_TOOL)
+            else:
+                rarfile._check_unrar_tool()
+        except (rarfile.RarCannotExec, rarfile.RarExecError, OSError):
+            if platform.system() == 'Windows':
+                logger.info('Looking for UNRAR in your installed programs')
+                winrar_path = 'WinRAR\\UnRAR.exe'
+                # Make a set of unique paths to check from existing environment variables
+                check_locations = {
+                    os.path.join(location, winrar_path) for location in (
+                        os.environ.get("ProgramW6432"), os.environ.get("ProgramFiles(x86)"),
```

Comment:
Why can't I ? 
As I'm a contributor I'm following 
https://github.com/pymedusa/Medusa/blob/master/CONTRIBUTING.md and it says nothing about it. Please update contribution.md updated if that is the case